main.go

```diff
@@ -79,6 +79,7 @@ func main() {
 	if printConfig {
 		configurationString, err := configuration.String()
 		if err != nil {
+			fmt.Println(err)
```

Comment:
Please make all outputs on this level a log output like 

```go
log.Printf("Failed to convert configuration to string: %+v\n", err)
``` 

The `"%+v"` will print a complete error backtrace which very handy.